### PR TITLE
Make Sandbox Great Again

### LINKED ISSUE #9: Missing dependencies when running unit tests
I had to install the following RPMS in order to run the Server/run-tests.sh script in the develop branch.

```# yum install python2-mock python-pwquality python2-qpid-proton```

### LINKED ISSUE #10: Sandbox is not working out of the box
At this moment we are providing 2 times of playbooks.
One is called setup_beaker and the second one setup_sandbox.

Setup_beaker is focused more on the full experience of beaker and setup_sandbox is more suitable for developers. Right now setup_beaker is working fine, however, due to changes for setup_beaker we broke setup_sandbox. This needs to be addressed.

### PR DESCRIPTION
**Pull request by StykMartin**
- Add autostart variable into variables
- IP range changed because it may collide with the CRC range. 

Fixes #9 
Fixes #10 
Signed-off-by: Martin Styk <mastyk@redhat.com>

### CODE REVIEW

File: roles/beaker/sandbox/packages/tasks/main.yml [hunk 1]
```diff
@@ -45,33 +45,16 @@
...
-    dest: /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-Cloud
-    owner: root
-    group: root
-    mode: 0600
-
-- name: import CentOS Cloud SIG key
-  rpm_key:
+- name: Install SIG Cloud OpenStack
+  package:
+    name: centos-release-openstack-train
```
Comment (StykMartin) [inline roles/beaker/sandbox/packages/tasks/main.yml:51]: Nice choice :rocket:

File: roles/beaker/sandbox/packages/tasks/main.yml [hunk 2]
```diff
@@ -9,42 +9,41 @@
...
   package:
-    name: "{{ item }}"
+    name:
+        - python-gunicorn
+        - nodejs-less
+        - syslinux-tftpboot
+        - libvirt-client
+        - firefox # for tests
+        - openldap-servers
+        - python-gunicorn
```
Comment (StykMartin) [inline roles/beaker/sandbox/packages/tasks/main.yml:19]: We need one more
**python-gssapi**.
It is necessary for common

File: roles/beaker/sandbox/packages/tasks/main.yml (hunk 1 above)
Comment (renanrodrigo) [reply roles/beaker/sandbox/packages/tasks/main.yml:51]: I wonder how I came up with this

File: roles/beaker/sandbox/packages/tasks/main.yml (hunk 2 above)
Comment (renanrodrigo) [reply roles/beaker/sandbox/packages/tasks/main.yml:19]: there it is

**Review by renanrodrigo — APPROVED** [review]
plusoned

**Comment by StykMartin** [comment]
@renanrodrigo thank you. Great collaboration mate!

**Pull request merged**